run_unittests.py

```diff
@@ -4125,6 +4125,21 @@ def test_check_module_linking(self):
                'recommended as it is not supported on some platforms')
         self.assertIn(msg, out)
 
+    def test_mixed_language_linker_check(self):
+        testdir = os.path.join(self.unit_test_dir, '97 compiler.links file arg')
+        self.init(testdir)
+        cmds = self.get_meson_log_compiler_checks()
+        self.assertEqual(len(cmds), 5)
+        cc = cmds[0][0]
+        cxx = cmds[1][0]
+        if cc == cxx:
+            raise unittest.SkipTest(f'Using a single compiler driver for both C and C++')
```

Comment:
is there really any reason not to test the cc == cxx path?

Reply:
I was thinking that in principle you do not need the separate compiler/linker execution if cc == cxx. But in that case the test would fail because len(cmds) == 4; so I just removed the if.